open-reac/src/main/java/com/powsybl/openreac/parameters/input/json/OpenReactJsonModule.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac.parameters.input.json;
+
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.powsybl.openreac.parameters.input.OpenReacParameters;
+import com.powsybl.openreac.parameters.input.VoltageLimitOverride;
+import com.powsybl.openreac.parameters.input.algo.OpenReacAlgoParam;
+
+/**
+ * @author Hugo Marcellin <hugo.marcelin at rte-france.com>
```

Comment:
Wrong author ? (on several files)

Reply:
this is copied from gridsuite so I kept original author